WeakAuras.lua

```diff
@@ -1810,6 +1810,28 @@ function WeakAuras.Add(data)
   -- end
 end
 
+
+function removeSpellNames(data)
```

Comment:
WA.lua is a 2-spaces indented file, might have to change that here, too when you paste it in